generator/source/faq/videos.rst

```diff
@@ -31,6 +31,12 @@ Intel: Building Blockchain for the Enterprise. Shows how Sawtooth builds are don
 Sawtooth 1.0 Architecture and App Development (31:26, Zac Delventhal, Bitwise, 2018)
     https://www.youtube.com/watch?v=uBebFQM49Xk
 
+Hyperledger Sawtooth Technical Overview Development (58:50, Dan Anderson, Intel, 2018)
```

Comment:
Delete "Development" (the title in the video is "Hyperledger Sawtooth Technical Overview").

Reply:
OK, thanks.